### ISSUE TITLE
spheregroup failing simple example

### ISSUE THREAD

**Issue by moustakas** [weaverba137/pydl#5]
The spheregroup module in pydl is crashing at multiple places.  The first one (based on the master branch version of the repository) is reproduced below, which can be easily fixed by changing "warning" to "warnings".  However, the subsequent crashes (with the same example) happen at increasingly deeper parts of the code.  Any help would be greatly appreciated!

```
import numpy as np
from pydl.pydlutils.spheregroup import spheregroup
ra = np.random.uniform(0,1,100)
dec = np.random.uniform(0,1,100)
gg = spheregroup(ra,dec,10.0/3600.0)
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-7-91139cb89eb3> in <module>()
----> 1 gg = spheregroup(ra,dec,10.0/3600.0)

/usr/local/lib/python2.7/site-packages/pydl-0.4.0.dev377-py2.7.egg/pydl/pydlutils/spheregroup/spheregroup.pyc in spheregroup(ra, dec, linklength, chunksize)
     37     from .. import PydlutilsUserWarning
     38     from numpy import zeros
---> 39     from warning import warn
     40     npoints = ra.size
     41     if npoints == 1:

ImportError: No module named warning
```

**Comment by weaverba137** [comment]
I have fixed the warnings problem, & I am developing a test suite for spheregroup.  I've fixed a few more simple errors "further down".

**Comment by weaverba137** [comment]
This should be fixed in the latest master revision.  Please try it out.

**Comment by moustakas** [comment]
This looks great!  I did a number of simple tests and also compared the outputs against the IDL/C version of the code, and everything matched beautifully.

**Comment by weaverba137** [comment]
Good.  The fixes will be incorporated into pydl-0.4.0, which should be released before the end of July.
